### DISCUSSION TITLE
【探索委托】找回旧时代的宝藏帖子

### DISCUSSION THREAD

**Discussion by BJ-cmyk** [share121/inter-knot#4756] [委托]
![1723465127837](https://github.com/user-attachments/assets/dc8a6bf8-a176-4bba-b6f3-c4354e3c5d50)
两周不来绳网，这给我干哪来了？之前的帖子呢？
求传奇绳匠接委托，帮我寻找消失的宝藏帖子

**Comment by sensei-is-playing-GenShin** [comment]
这个委托能给多少青辉石？

**Comment by Yizakl** [comment]
被简笔删辣
